### ISSUE TITLE
static method of user model requires user_id

### ISSUE THREAD

**Issue by patvdleer** [thomaxxl/safrs#28]
I would expect this not to be the case since it is a `staticmethod`. I looked at the search option, see below which one I mean. This is build on a `classmethod`. 

```python
@classmethod
@jsonapi_rpc(http_methods=['POST'])
def search(cls, **kwargs):
```

my code

```python
class User(Model):
    custom_decorators = [jwt_required, auth.login_required]
    id = db.Column(db.Integer, primary_key=True)
    name = db.Column(db.String(255))
    email = db.Column(db.String(80, collation='NOCASE'), unique=True, nullable=False)
    password_hash = db.Column(db.String(128))
    language = db.Column(db.String(5), nullable=True, default="en")
    timezone = db.Column(db.String(64), nullable=True, default="Europe/Amsterdam")

    @staticmethod
    @jsonapi_rpc(http_methods=['POST'])
    def login(email_or_token, password):
        user = User.verify_auth_token(email_or_token)
        if not user:
            user = User.query.filter_by(email=email_or_token).first()
            if not user or not user.verify_password(password):
                return None
        return user.create_access_token()
```

![image](https://user-images.githubusercontent.com/1138136/54214422-7bb2e800-44e6-11e9-9829-f3e355228758.png)

**Comment by thomaxxl** [comment]
I never considered staticmethods, It's implemented in ca8a28ef0891cfcb75933d8ea92b00dc768b683c now :)

**Comment by patvdleer** [comment]
damn dude you're fast, awesome! thanks!
